financial-templates-lib/test/ExpiringMultiPartyClient.js

```diff
@@ -271,7 +251,110 @@ contract("ExpiringMultiPartyClient.js", function(accounts) {
     await emp.dispute(id.toString(), accounts[0], { from: accounts[0] });
     await client._update();
 
-    // The disputed liquidation should longer show up as undisputed.
+    // The disputed liquidation should no longer show up as undisputed.
+    assert.deepStrictEqual([], client.getUndisputedLiquidations().sort());
+  });
+
+  it("Returns expired liquidations", async function () {
+    await emp.create({ rawValue: toWei("150") }, { rawValue: toWei("100") }, { from: accounts[0] });
+    await emp.create({ rawValue: toWei("1500") }, { rawValue: toWei("100") }, { from: accounts[1] });
+
+    // Create a new liquidation for account[0]'s position.
+    await emp.createLiquidation.call(
+      accounts[0],
+      { rawValue: toWei("9999999") },
+      { rawValue: toWei("150") },
+      { from: accounts[1] }
+    );
+    await emp.createLiquidation(
+      accounts[0],
+      { rawValue: toWei("9999999") },
+      { rawValue: toWei("150") },
+      { from: accounts[1] }
+    );
+    await client._update();
+
+    const liquidations = client.getUndisputedLiquidations();
+    const liquidationTime = liquidations[0].liquidationTime;
+    assert.deepStrictEqual(
+      [
+        {
+          sponsor: accounts[0],
+          id: "0",
+          liquidationTime: liquidationTime,
+          numTokens: toWei("100"),
+          amountCollateral: toWei("150")
+        }
+      ],
+      liquidations
+    );
+    assert.deepStrictEqual([], client.getExpiredLiquidations().sort());
+
+    // Move EMP time to the liquidation's expiry.
+    const liquidationLiveness = 1000;
+    await emp.setCurrentTime(Number(liquidationTime) + liquidationLiveness);
+    await client._update();
+
+    // The liquidation is registered by the EMP client as expired.
     assert.deepStrictEqual([], client.getUndisputedLiquidations().sort());
```

Comment:
Per my above comment, here we test that liquidations are removed from undisputedLiquidations once they expire.